build/build-plugins.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Script to build our plugins to use them separately.
+ * Copyright 2017-2018 The Bootstrap Authors
+ * Copyright 2017-2018 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+'use strict'
+
+const rollup  = require('rollup')
+const path    = require('path')
+const babel   = require('rollup-plugin-babel')
+
+const plugins = [
+  babel({
+    exclude: 'node_modules/**', // Only transpile our source code
+    externalHelpersWhitelist: [ // Include only required helpers
+      'defineProperties',
+      'createClass',
+      'inheritsLoose',
+      'defineProperty',
+      'objectSpread'
+    ]
+  })
+]
+
+const format = 'umd'
+const bsPlugins = {
+  Alert: path.resolve(__dirname, '../js/src/alert.js'),
+  Button: path.resolve(__dirname, '../js/src/button.js'),
+  Carousel: path.resolve(__dirname, '../js/src/carousel.js'),
+  Collapse: path.resolve(__dirname, '../js/src/collapse.js'),
+  Dropdown: path.resolve(__dirname, '../js/src/dropdown.js'),
+  Modal: path.resolve(__dirname, '../js/src/modal.js'),
+  Popover: path.resolve(__dirname, '../js/src/popover.js'),
+  ScrollSpy: path.resolve(__dirname, '../js/src/scrollspy.js'),
+  Tab: path.resolve(__dirname, '../js/src/tab.js'),
+  Tooltip: path.resolve(__dirname, '../js/src/tooltip.js'),
+  Util: path.resolve(__dirname, '../js/src/util.js')
+}
+
+const defaultExternal = ['jquery', 'popper.js']
+const defaultGlobals = {
+  jquery: 'jQuery', // Ensure we use jQuery which is always available even in noConflict mode
+  'popper.js': 'Popper'
+}
+
+Object.keys(bsPlugins)
+  .forEach((pluginKey) => {
+    console.log(`Building ${pluginKey} plugin...`)
+
+    const external = [].concat(defaultExternal)
+    const globals = {
+      ...defaultGlobals
+    }
+
+    // Do not bundle Util in plugins
+    if (bsPlugins[pluginKey] !== bsPlugins.Util) {
+      external.push(bsPlugins.Util)
+      globals['./util'] = 'Util'
+    }
+
+    // Do not bundle Tooltip in Popover
+    if (bsPlugins[pluginKey] === bsPlugins.Popover) {
```

Comment:
woudln't `pluginKey === 'Popover'` be simpler ?

Reply:
Yep it'll clearer thanks @stof 👍